README.md

```diff
@@ -63,53 +63,50 @@ confluent consume test1 --value-format avro --max-messages 5 --property print.ke
 ```
+This will build the connector from source and create a local image with an aggregate version number.  The aggregate version number is the kafka-connect-datagen connector version number and the Confluent Platform version number separated with a `-`.   The local kafka-connect-datagen version number is defined in the `pom.xml` file, and the Confluent Platform version defined in the `Makefile` (example `0.1.5-5.3.1`).
 
-### Build connector from latest code
+### Install connector from Confluent Hub
 
-Alternatively, you may build and install the `kafka-connect-datagen` connector from latest code.
-Here we use `v0.1.5` to reference the git tag for the `0.1.5` version, but the same pattern works for all released versions.
-Be sure to use the same version in the Docker image tag (e.g., `confluentinc/kafka-connect-datagen:0.1.5`) that you checked out (e.g., `v0.1.5`).
-
-```bash
-git checkout v0.1.5
-mvn clean package
-docker build . -f Dockerfile-local -t confluentinc/kafka-connect-datagen:0.1.5
-```
+You may install into your Docker image the `kafka-connect-datagen` connector from [Confluent Hub](https://www.confluent.io/connector/kafka-connect-datagen/).  The `Dockerfile-confluenthub` file has an example of how this can be accomplished.  The `Makefile` has targets to build an image from the standard `cp-kafka-connect` and the `cp-server-connect-operator` base image if running Confluent Platform on Kubernetes.
 
 ### Run connector in Docker Compose
 
 Here is an example of how to run the `kafka-connect-datagen` with Docker Compose.
-If you used a different Docker image tag, be sure to use that here instead of `confluentinc/kafka-connect-datagen:0.1.5`.
+If you wish to use a different Docker image tag, be sure to modify appropriately in the `docker-compose.yml` file.
 
 ```bash
 docker-compose up -d --build
 curl -X POST -H "Content-Type: application/json" --data @config/connector_pageviews.config http://localhost:8083/connectors
 docker-compose exec connect kafka-console-consumer --topic pageviews --bootstrap-server kafka:29092  --property print.key=true --max-messages 5 --from-beginning
 ```
 
-### Publish Docker Image
+### Publish Docker Images
+
+Note: The provided Makefile and `make publish` command publishes the Docker image to https://hub.docker.com/r/cnfldemos/kafka-connect-datagen/ , which is accessible only to the organization admins. The local Docker daemon must be logged into a proper Docker Hub account.
 
-To release a new version and publish a new Docker image to Dockerhub at https://hub.docker.com/r/cnfldemos/kafka-connect-datagen/:
+To release new versions of the Docker images to Dockerhub (https://hub.docker.com/r/cnfldemos/kafka-connect-datagen/ & https://hub.docker.com/r/cnfldemos/cp-server-connect-operator-with-datagen)
 
-* Update the version numbers in `Dockerfile-dockerhub`
-* Run the following:
+The `Makefile` contains a publish target and you can override the version numbers used by overriding the following variables:
```

Comment:
Tell users where to find the default values

Reply:
Are you thinking pointing to line numbers?  Or figuring out how to do a permalink into the file from this doc?  The Makefile is pretty concise and follows a common pattern of variables near the top, it feels relatively self documenting in this regard IMO.

Reply:
I added a small blurb about variable names, defaults and approximate location in the README